pkg/sql/sqlbase/testutils.go

```diff
@@ -796,21 +809,25 @@ func RandCollationLocale(rng *rand.Rand) *string {
 
 // RandType returns a random type value.
 func RandType(rng *rand.Rand) *types.T {
-	return randType(rng, seedTypes)
+	return RandTypeFromSlice(rng, SeedTypes)
 }
 
+// Defeat the linter.
+var _ = RandScalarType
```

Comment:
Just remove this function if it's not used anymore. The change to RandTypeFromSlice means we don't need it now.